src/clparser_perftest.cc

```diff
@@ -146,7 +146,7 @@ int main(int argc, char* argv[]) {
     int64_t end = GetTimeMillis();
 
     if (end - start > 2000) {
-      int delta_ms = (int)(end - start);
+      int delta_ms = static_cast<int>(end - start);
```

Comment:
int64_t - int64_t is not int. Please ensure the result is stored as the proper accuracy.

Reply:
Because there are lots of other more important things to improve in Ninja.

Reply:
I would merge the static_cast change without changing the type to int64_t, all I'm saying.

Reply:
Ok. static_cast<int> is better than (int), that I agree with.

I would still prefer that if we're going to change anything at all, we do it "right" immediately though.